src/logics/operate.js

```diff
@@ -0,0 +1,26 @@
+const Big = require('big.js');
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = 0;
+  switch (operation) {
+    case '+':
+      result = Big(numberOne) + Big(numberTwo);
+      break;
+    case '-':
+      result = Big(numberOne) - Big(numberTwo);
+      break;
+    case '*':
+      result = Big(numberOne) * Big(numberTwo);
+      break;
+    case '/':
+      result = Big(numberOne) / Big(numberTwo);
+      break;
+    case '%':
```

Comment:
You forgot to manage division by ZERO 😊

Reply:
Hello I have a question.
Handling of divivsion by zero wont it come at the component level depending on what i recieve from the logic bit.

Reply:
Cause i cant(not suppose to) create alert in the logic section of the project.

Reply:
@Urchmaney  Oh yeah totally bro, you can handle it there, was just drawing your attention

Reply:
ok thanks.